code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -10,7 +10,7 @@
 	throwforce = 0
 	throw_speed = 3
 	throw_range = 6
-	grind_results = list()
+	grind_results = list(/datum/reagent/toxin/slimejelly = 20)
 	///uses before it goes inert
 	var/extract_uses = 1
```

Comment:
You can remove this var entirely. Or does it do something else?

Reply:
...yes, grind_results being set is the reason as to why this fix works. It cannot be removed.

Reply:
Not grind results. I was talking about extract_uses

Reply:
> Not grind results. I was talking about `extract_uses`

...Yes, that is also used. Cerulean slimes can upgrade slime extracts to make them have more uses.